truffle/test/VaultNotaryTest.js

```diff
@@ -0,0 +1,384 @@
+const truffleAssert = require('truffle-assertions');
+const uuidv4 = require('uuid/v4');
+const uuidToHex = require('uuid-to-hex');
+
+const NotaryContract = artifacts.require("VaultNotary");
+
+
+
+function uuidToHex32(uuid) {
+    return uuid + '00000000000000000000000000000000'
+}
+
+
+contract('VaultNotary', async (accounts) => {
+    const OWNER = accounts[0];
+    const SHIPPER = accounts[1];
+    const CARRIER = accounts[2];
+    const MODERATOR = accounts[3];
```

Comment:
I think since this is separate from the LOAD contract we shouldn't use the shipper/carrier/moderator terminology. Maybe OWNER/ALICE/BOB/ATTACKER is sufficient?